java-parent-pom/pom.xml

```diff
@@ -24,7 +24,7 @@
     <parent>
         <groupId>net.openhft</groupId>
         <artifactId>root-parent-pom</artifactId>
-        <version>1.2.8</version>
+        <version>1.2.9-SNAPSHOT</version>
```

Comment:
You know root-parent-pom needs to be released manually. I don't think this will fly with the release process.